lib/main_containers.pm

```diff
@@ -146,11 +153,12 @@ sub load_host_tests_docker {
     unless (is_generalhw || is_ipmi || is_public_cloud || is_openstack || is_sle_micro || is_microos || is_leap_micro) {
         loadtest 'containers/validate_btrfs';
     }
-    load_volume_tests($run_args);
     if (is_tumbleweed || is_microos) {
         loadtest 'containers/buildx';
         loadtest 'containers/rootless_docker';
     }
+    load_secret_test($run_args);
+    load_volume_tests($run_args);
```

Comment:
is there any specific reason why you moved the volume test?

Reply:
@ricardobranco777 suggested me that he added some cleanup in his test that might be causing mine to fail, so we decided to swap the order and it started to work properly :)

Reply:
hm ... IMHO all of these container modules should also work as a stand alone test module. Thus the clean up of one should not affect yours